src/imgbased/plugins/service.py

```diff
@@ -56,8 +56,8 @@ def _generate_iqn(self):
             return
         log.debug("Description=Generate a random iSCSI initiator IQN name")
         suuid = str(uuid.uuid4()).split("-")[-1]
-        factory_f = File("/usr/share/factory/etc/iscsi/initiatorname.iscsi")
-        iqn = factory_f.contents.split(":")[0] + ":" + suuid + "\n"
+        identifier = command.call("iscsi-iname")
```

Comment:
LGTM, but I would suggest to use full path here to avoid any potential surprises

Reply:
Fixed!